packages/nimble-components/src/table/styles.ts

```diff
@@ -65,6 +66,14 @@ export const styles = css`
         left: var(--ni-private-table-scroll-x);
     }
 
+    .header-row::before {
```

Comment:
Is this styling of `.header-row::before` necessary? It isn't immediately clear to me what style it is providing since it doesn't seem to provided a new background color.